### PR TITLE
'Name' textfield on focus instead of 'OK' button when user clicks on 'Add subgroup' option

### PR DESCRIPTION
**Pull request by HussainArif12**
Fixes  : #6307
Name area on focus instead of button whenever user clicks on 'Add subgroup' option

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [x] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

I have addressed the changes. 
Also can you tell me how do we amend commits on git?
Thank you

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -246,6 +246,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added a 'LaTeX citations' tab to the entry editor, to search for citations to the active entry in the LaTeX file directory. It can be disabled in the preferences dialog.
 - We added an option in preferences to allow for integers in field "edition" when running database in bibtex mode. [#4680](https://github.com/JabRef/jabref/issues/4680)
 - We added the ability to use negation in export filter layouts. [#5138](https://github.com/JabRef/jabref/pull/5138)
+-Focus on Name Area instead of 'OK' button whenever user presses 'Add subgroup'
```
Comment (calixtus) [inline CHANGELOG.md:249]: Missing whitespace

File: src/main/java/org/jabref/gui/groups/GroupDialogView.java [hunk 2]
```diff
@@ -97,7 +97,7 @@ public void initialize() {
         hierarchyText.put(GroupHierarchyType.INDEPENDENT, Localization.lang("Independent"));
         hierarchyToolTip.put(GroupHierarchyType.INDEPENDENT, Localization.lang("Independent group: When selected, view only this group's entries"));
 
-        nameField.textProperty().bindBidirectional(viewModel.nameProperty());
+         nameField.textProperty().bindBidirectional(viewModel.nameProperty());
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/groups/GroupDialogView.java:100]: Indent is off again

File: src/main/java/org/jabref/gui/groups/GroupDialogView.java [hunk 3]
```diff
@@ -131,7 +131,7 @@ public void initialize() {
         autoGroupPersonsField.textProperty().bindBidirectional(viewModel.autoGroupPersonsFieldProperty());
 
         texGroupFilePath.textProperty().bindBidirectional(viewModel.texGroupFilePathProperty());
-
+
```
Comment (calixtus) [inline src/main/java/org/jabref/gui/groups/GroupDialogView.java:134]: Whitespaces are superflous

**Comment by Siedlerchr** [comment]
I always use the simple git gui. There is a button amend last commit. But if you have pushed that commit already, you need to do a force push then. Otherwise just do a new commit with the changes. In the end we squash them on merge, so they will be all one commit.

**Comment by HussainArif12** [comment]
Committed new change. Please check

File: CHANGELOG.md [hunk 4]
```diff
@@ -246,6 +246,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added a 'LaTeX citations' tab to the entry editor, to search for citations to the active entry in the LaTeX file directory. It can be disabled in the preferences dialog.
 - We added an option in preferences to allow for integers in field "edition" when running database in bibtex mode. [#4680](https://github.com/JabRef/jabref/issues/4680)
 - We added the ability to use negation in export filter layouts. [#5138](https://github.com/JabRef/jabref/pull/5138)
+- Focus on Name Area instead of 'OK' button whenever user presses 'Add subgroup'
```
Comment (Siedlerchr) [inline CHANGELOG.md:249]: Please also add a link to the issue like in the other lines

**Comment by HussainArif12** [comment]
Changelog.md is now fixed in new commit

**Comment by Siedlerchr** [comment]
Thank you very much for your contribution! 👍

**Pull request merged**